src/api/database/models/User.ts

```diff
@@ -69,6 +73,41 @@ class User extends ParanoidModel<
 
   @HasMany(() => Interview)
   interviews: NonAttribute<Interview[]>;
+
+  @BeforeDestroy
+  static async cascadeDelete(user: User, options: any) {
+
+    if (options.force) {
```

Comment:
why do you need to distinguish forced and non-forced deletion? can you just do:

```
await GroupUser.destroy({
        where: { userId: user.id },
        ...options,
      });
```

in both cases? same for partnership.